### PR TITLE
Added Target "IFRC-IFLIGHT_F411_AIO32"

### PR DESCRIPTION
**Pull request by Linjieqiang**
image
image

### CODE REVIEW

File: configs/default/IFRC-IFLIGHT_F411_AIO32.config [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature SOFTSERIAL
+feature TELEMETRY
+feature OSD
+feature AIRMODE
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F411_AIO32.config:89]: Please remove all settings that are defaults - see the instructions at https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md for a way to do this.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/IFRC-IFLIGHT_F411_AIO32.config (hunk 1 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F411_AIO32.config]: Okay. Thanks..

**Comment by Linjieqiang** [comment]
@mikeller Fixed it.

File: configs/default/IFRC-IFLIGHT_F411_AIO32.config [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ADC_BATT 1 B00
+resource ADC_CURR 1 B01
+resource FLASH_CS 1 A15
+resource OSD_CS 1 B12
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F411_AIO32.config:34]: Looks like now you have removed _all_ of the features - from the look of it this board does have an OSD chip, so `feature OSD` should be set to have this enabled by default (as per https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#26-add-custom-settings-for-your-board-to-the-unified-target-configuration).

Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F411_AIO32.config:34]: Okay. I will change it.

**Comment by Linjieqiang** [comment]
Forgive me for dealing with this matter so late. Now it has been repaired as required.

**Review by mikeller — APPROVED** [review]

**Pull request merged**